spec/game_spec.rb

```diff
@@ -0,0 +1,80 @@
+# rubocop:disable Layout/LineLength
+require './lib/board.rb'
+require './lib/logic.rb'
+
+describe Board do
+  describe '#display_board' do
+    it 'Returns the display of board' do
+      board = Board.new
+      expect(board.display_board([1, 2, 3, 4, 5, 6, 7, 8, 9])).to eql("  1  |  2  |  3  \n  4  |  5  |  6  \n  7  |  8  |  9  \n ")
+    end
+    it 'Returns the display of Empty board' do
+      board = Board.new
+      expect(board.display_board(['-', '-', '-', '-', '-', '-', '-', '-', '-'])).to eql("  -  |  -  |  -  \n  -  |  -  |  -  \n  -  |  -  |  -  \n ")
+    end
```

Comment:
It looks that both tests are checking for the same thing. That the input you give to the `display_board` method is the output between` | |` so I don't see the point of testing both.